x/erc20/types/events.go

```diff
@@ -11,22 +11,15 @@ import (
 
 // erc20 events
 const (
-	EventTypeTokenLock             = "token_lock"
-	EventTypeTokenUnlock           = "token_unlock"
-	EventTypeMint                  = "mint"
-	EventTypeConvertCoin           = "convert_coin"
-	EventTypeConvertERC20          = "convert_erc20"
-	EventTypeBurn                  = "burn"
-	EventTypeRegisterCoin          = "register_coin"
-	EventTypeRegisterERC20         = "register_erc20"
-	EventTypeToggleTokenConversion = "toggle_token_conversion" // #nosec
+	EventTypeConvertERC20           = "convert_erc20"
+	EventTypeRegisterERC20          = "register_erc20"
+	EventTypeToggleTokenConversion  = "toggle_token_conversion" // #nosec
```

Comment:
Q: what's being "#nosec"'d here? Would be good to add an explainer even though not necessarily related to this PR.

Reply:
It's added there because linters fail thinking it's a hardcoded credential